### PR TITLE
Bump babel packages to latest v6 versions

### PR DESCRIPTION
**Pull request by taymoork2**
Bump babel packages to latest v6 versions

## Type of Change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: .vscode/launch.json [hunk 1]
```diff
@@ -13,18 +13,45 @@
...
+        "${input:flags}"
+      ]
+    }
+  ],
+  "inputs": [
+    {
+        "id": "packages",
+        "type": "promptString",
+        "description": "Which package do you want to debug (i.e. `plugin-meetings`)"
+        // type specific configuration attributes
```
Comment (InteractiveTimmy) [inline .vscode/launch.json]: Don't think commenting in `.json` file is a good idea, as it's not a part of the **JSON** standard. correct me if this is actually okay by our standards.

File: .vscode/launch.json [hunk 2]
```diff
@@ -13,18 +13,45 @@
...
+    },
+    {
+        "id": "flags",
+        "type": "promptString",
+        "description": "Which flags would you like to add (i.e. `--node` or `--unit`)"
+        // type specific configuration attributes
```
Comment (InteractiveTimmy) [inline .vscode/launch.json]: Don't think commenting in `.json` file is a good idea, as it's not a part of the **JSON** standard. correct me if this is actually okay by our standards.

Comment (taymoork2) [reply .vscode/launch.json]: It’s .vscode’s
I think it comes with comments by default to explain everything

Comment (taymoork2) [reply .vscode/launch.json]: Ill remove them anyway

**Comment by taymoork2** [comment]
Merged

**Pull request closed without merging**